cmd/collection/main.go

```diff
@@ -100,7 +100,7 @@ func main() {
 				"maximum number of transactions in proposed collections")
 			flags.Uint64Var(&maxCollectionByteSize, "builder-max-collection-byte-size", 1000000,
 				"maximum byte size of the proposed collection")
-			flags.Uint64Var(&maxCollectionTotalGas, "builder-max-collection-total-gas", 1000000,
+			flags.Uint64Var(&maxCollectionTotalGas, "builder-max-collection-total-gas", 10000000,
```

Comment:
1 million was probably fine, but i think 10million should still be okay?

Reply:
yeah better to go with the bigger number than the smaller one for now